packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/DartMessenger.java

```diff
@@ -135,43 +102,8 @@ void sendCameraClosingEvent() {
    * @param description contains details regarding the error that occurred.
    */
   void sendCameraErrorEvent(@Nullable String description) {
-    this.send(
-        CameraEventType.ERROR,
-        new HashMap<String, Object>() {
-          {
-            if (!TextUtils.isEmpty(description)) put("description", description);
-          }
-        });
-  }
-
-  private void send(CameraEventType eventType) {
-    send(eventType, new HashMap<>());
-  }
-
-  private void send(CameraEventType eventType, Map<String, Object> args) {
-    if (cameraChannel == null) {
-      return;
-    }
-
-    handler.post(
-        new Runnable() {
-          @Override
-          public void run() {
-            cameraChannel.invokeMethod(eventType.method, args);
-          }
-        });
-  }
-
-  private void send(DeviceEventType eventType) {
-    send(eventType, new HashMap<>());
-  }
-
-  private void send(DeviceEventType eventType, Map<String, Object> args) {
-    if (deviceChannel == null) {
-      return;
-    }
-
-    handler.post(() -> deviceChannel.invokeMethod(eventType.method, args));
+    String errorMessage = (description == null) ? "" : description;
```

Comment:
Could you put a TODO (with an issue link) on this line to fix `description` being nullable and update any failing call sites? Actually fixing it is out of scope of this PR, but the fact that we need this line is a significant problem since there's no other content in the message, so if we ever actually do this, the plugin client will get absolutely no actionable information.